### DISCUSSION TITLE
Course discussions

### DISCUSSION THREAD

**Discussion by cmwilson21** [skills/.github#311] [Announcements]
Course discussions are now available for the Skills community! 🎉 

From the Discussions page, scroll down to the section "Skills courses" on the left and expand it to see all of the Skills courses currently available. Click into the course section to ask questions, start conversations, and share your feedback!

Thanks for continuing to support each other throughout the courses 💖

**Comment by AniketNS** [comment]
While following the course, I'm at the step where I've created the codespace and now they've told me to refresh the repository landing page after 60 seconds. But there nothing is coming after 60 seconds when I refreshed that page.

![image](https://github.com/skills/.github/assets/91371143/15c8c2d4-19b0-4b79-a48d-d4688f93e1b7)

**Reply by Godie360** [reply]
Do it agin

**Comment by Lumiere-MULAGWA** [comment]
vFrom the Discussions page, scroll down to the section "Skills courses" on the left and expand it to see all of the Skills courses currently available. Click into the course section to ask questions, start conversations, and share your feedback!

Thanks for continuing to support each other throughout the courses 💖
